<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test9_1AClass</class>
 <widget class="QMainWindow" name="Test9_1AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test9_1A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>用户设置</string>
         </attribute>
         <widget class="QLineEdit" name="txtPwd">
          <property name="geometry">
           <rect>
            <x>78</x>
            <y>90</y>
            <width>129</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>53</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>密  码：</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>187</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>用户名：</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="txtName"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>系统设置</string>
         </attribute>
         <widget class="QSplitter" name="splitter_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>95</width>
            <height>32</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QCheckBox" name="cbxSaveUserInfo">
           <property name="text">
            <string>保存用户信息</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbxAutoLogin">
           <property name="text">
            <string>自动登录</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>498</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>保存设置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Test9_1A.qrc"/>
 </resources>
 <connections/>
</ui>
